Libraries/Components/TextInput/TextInput.js

```diff
@@ -673,6 +690,12 @@ const TextInput = React.createClass({
     }
   },
 
+  _onContentSizeChange(event: Event) {
+    if (this.props.onContentSizeChange) {
+      this.props.onContentSizeChange(event.nativeEvent.contentSize);
```

Comment:
This seems unlike any of the other event handlers we provide. If we add a new property to this event, it's really hard to make it backwards compatible.